02-Reproject-Raster-In-R.Rmd

```diff
@@ -194,32 +205,34 @@ crs(DTM_hill_HARV)
 extent(DTM_hill_UTMZ18N_HARV)
 extent(DTM_hill_HARV)
 
-
 ```
 
 Notice in the output above that the CRS of `DTM_hillUTMZ18N_HARV` is now UTM. 
 However, the extent of the object is different. 
+
 Why do you think this is?  
 
+Note: When you are reprojecting a raster, you are moving it from one "grid" 
```

Comment:
{: .notice1} is text with a non-shadowed, grey box. It is used in other NEON Data Skills lessons so I think it should be retained and is useful when something needs to stand out VERY separate from the lessons.  However, as I read through lessons I find that this breaks the lesson up, setting the information apart from the lesson instead of as an integral part of the lesson, which it is.  
I'd started using the > at the start of the line as I found the formatting of a solid grey bar on both sides of the text a better way to highlight the Note but not separate it form the lesson.  However, as you suggested, having > at the start of each line makes for messy code.  You suggested a {: .notice2} could be created with that format to make for neater coding.